### PR TITLE
Minor tweaks to index.md

### PR DESCRIPTION
**Pull request by samdutton**
Good post!

I've done these suggestions as a PR, but you may want to ignore all or part of this. Quite fiddly to do this as individual (multiline) suggestions.

BTW: ` The contents of this method should aim to get access to the stream source` sounds odd to me. I've tweaked the text, but not sure if I've got it quite right.

BTW 2: love the use of `1_000`, `10_000`, etc.!

Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

- 
- 
-

### CODE REVIEW

**Review by mihajlija — APPROVED** [review]

**Comment by kaycebasques** [comment]
@tomayac Sam is requesting to merge changes into your streams PR here

File: src/site/content/en/blog/streams/index.md [hunk 1]
```diff
@@ -138,24 +138,25 @@ reading your stream, you typically need to **cancel** the first reader before yo
...
+  method can access the stream source, and do anything else
+  required to set up the stream functionality. If this process is to be done asynchronously, the method can
   return a promise to signal success or failure. The `controller` parameter passed to this method is
   a
   [`ReadableStreamDefaultController`](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStreamDefaultController)
   by default.
-- `pull(controller)`: This method can be used to control the stream as more chunks are fetched. It
-  will be called repeatedly when the stream's internal queue of chunks is not full, up until it
+- `pull(controller)`: Can be used to control the stream as more chunks are fetched. It
+  is called repeatedly as long as the stream's internal queue of chunks is not full, up until queue
```
Comment (tomayac) [inline src/site/content/en/blog/streams/index.md:156]:
```suggestion
  is called repeatedly as long as the stream's internal queue of chunks is not full, up until the queue
```

**Review by tomayac — APPROVED** [review]

**Comment by kaycebasques** [comment]
Hello! This is an automated review by our custom [reviewbot](https://github.com/kaycebasques/reviewbot). It updates automatically when code or GitHub comments in this pull request are created or updated.

## Requested changes

If there are any common problems with the content files you created or modified, they will be listed here.

### `src/site/content/en/blog/streams/index.md`

* This file passed all of our automated Markdown audits.

**Comment by tomayac** [comment]
> I've done these suggestions as a PR, but you may want to ignore all or part of this. Quite fiddly to do this as individual (multiline) suggestions.

Thanks, @samdutton! <3 

> BTW: ` The contents of this method should aim to get access to the stream source` sounds odd to me. I've tweaked the text, but not sure if I've got it quite right.

You did, thanks for catching this!

> BTW 2: love the use of `1_000`, `10_000`, etc.!

It's one of these small things…

**Pull request merged**